GNUmakefile

```diff
@@ -5,6 +5,9 @@ default: test
 testall:
 	TF_ACC=1 go test ./... -v $(TESTARGS) -timeout 120m
 
+docker-run:
+	docker compose up -d --build
```

Comment:
nice :) TIL about docker compose vs docker-compose

Reply:
Yeah they added it not too long ago, nice to have it part of the docker binary. :slightly_smiling_face: